source/github_ee/github_ee_test.go

```diff
@@ -0,0 +1,36 @@
+package github_ee
+
+import (
+	"net/http"
+	"net/http/httptest"
+	nurl "net/url"
+	"strings"
+	"testing"
+)
+
+func Test(t *testing.T) {
+	ts := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		p := strings.Join(strings.Split(r.URL.Path, "/")[:3], "/")
+
+		if p != "/api/v3" {
+			t.Fatal()
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte("[]"))
+	}))
+	defer ts.Close()
+
+	u, err := nurl.Parse(ts.URL)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	g := &GithubEE{}
+	_, err = g.Open("github-ee://foo:bar@" + u.Host + "/mattes/migrate_test_tmp/test?skipSSLVerify=true#452b8003e7")
```

Comment:
Get the response and check that the status code is `http.StatusOK`

Reply:
`Open()` returns `Driver` and `error` types, so there isn't really a way to check for `http.StatusOK`. If `err` is `null`, the migrations then run, but that's tested in the `github` source driver. Unless I misunderstood the suggestion?